http/1.6/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -777,59 +710,54 @@ Set `tlsContext` to reference a TLS context element, provide your trust store an
 ----
 
-The `tlsContextFactory` attribute in the `oauth:authorization-code-grant-type` element points
-to the `<tls:context` element where you encode your OAuth credentials.
 
 [[oauth2cc]]
-== OAuth2 - Client Credentials
-
-In the OAuth authentication, the Client Credentials tab lets you configure the client credentials grant type.
+== Configure OAuth2 Client Credentials Grant Type Authentication
 
 The OAuth authentication server (OAS) is a server that holds the resources that are protected by OAuth. For example, the Box server provides an API with OAuth authentication.
 
-The client application (CA) is the server that tries to access a protected resource that belongs to a resource owner and that is held in an OAuth authentication server. For example, a Mule Server trying to access the resources that belong to a Box user and that are held in a Box server.
+The client application (CA) is the server that tries to access a protected resource that belongs to a resource owner (RO). For example, a Mule server tries to access the resources that belong to a Box user, and the resources are in a Box server.
 
-In this case, the resource owner (RO) is also the CA. This means that the CA is implicitly authorized by the RO, which makes the whole procedure a lot simpler.
+In this case, the RO is also the CA. This means that the CA is implicitly authorized by the RO and makes the whole procedure a lot simpler.
 
-image::oauth-danceposta-simple.png[]
+The following diagram describes the relationship between both CA and OAS:
+
+image::http-authentication-diagram-4.png[OAuth2 dance between CA and OAS]
 
 To access protected resources:
 
-. The CA must register an app to the OAS server. When this happens, the OAS assigns credentials to the CA that it can later use to identify itself: `client ID` and `client secret`. The OAS must also provide a `Token URL`, to which the CA can later send HTTP requests to retrieve an `access token` that is required when accessing the Protected Resources.
+. The CA must register an app to the OAS server. When this occurs, the OAS assigns credentials to the CA that it can later use to identify itself: `client ID` and `client secret`. The OAS must also provide a `Token URL`, to which the CA can later send HTTP requests to retrieve an `access token` that is required when accessing the protected resources.
 . The CA makes a request to the `Token URL` of the OAS, containing its client ID to prove its identity. As a response, the OAS grants it an `access token`.
-. With this access token, the CA is now free to access the protected resources in the OAS as long as it includes it in its requests. Depending on the policies defined by the OAS, this token may eventually expire.
-
-=== OAuth2 - Configuration
+. With the access token, the CA is now free to access the protected resources in the OAS as long as it includes the access token in the requests. Depending on the policies defined by the OAS, the token may eventually expire.
```

Comment:
Per above:
When the CA makes a request to the `Token URL` of the OAS, containing its client ID to prove its identity, the OAS grants the CA an `access token`. With the access token, the CA can access the protected resources in the OAS by including the access token in its requests. Depending on the policies defined by the OAS, the token eventually expires.